sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1429 @@
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1670412506460
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Get the AutoML best child run"
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Get the best model's child run\r\n",
+    "best_child_run_id = mlflow_parent_run.data.tags[\"automl_best_child_run_id\"]\r\n",
+    "print(\"Found best child run id: \", best_child_run_id)\r\n",
+    "\r\n",
+    "best_run = mlflow_client.get_run(best_child_run_id)\r\n",
+    "\r\n",
+    "print(\"Best child run: \")\r\n",
+    "print(best_run)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1670412507109
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 6. Model Evaluation and Forecasting"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 6.1 Forecasting using batch endpoint"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now that we have retrieved the best pipeline/model, it can be used to make predictions on test data. We will do batch inferencing on the test dataset which must have the same schema as training dataset.\n",
+    "\n",
+    "The inference will run on a remote compute. In this example, it will re-use the training compute."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### 6.1.1 Create a model endpoint\n",
+    "First, we need to register the model, environment and the batch endpoint."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "from azure.ai.ml.entities import (\n",
+    "    Environment,\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    BatchRetrySettings,\n",
+    "    Model,\n",
+    ")\n",
+    "from azure.ai.ml.constants import BatchDeploymentOutputAction\n",
+    "\n",
+    "model_name = \"forecast-recipe-univariate-run\"\n",
+    "batch_endpoint_name = \"forecast-recipe-univariate-run\"\n",
+    "\n",
+    "model = Model(\n",
+    "    path=f\"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/model.pkl\",\n",
+    "    name=model_name,\n",
+    "    description=\"Forecasting Recipe Univariate Model\",\n",
+    ")\n",
+    "registered_model = ml_client.models.create_or_update(model)\n",
+    "\n",
+    "env = Environment(\n",
+    "    name=\"automl-tabular-env\",\n",
+    "    description=\"environment for automl inference\",\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    conda_file=\"artifact_downloads/outputs/conda_env_v_1_0_0.yml\",\n",
```

Comment:
@Nivi09 

You need to download the artifacts from the best run, this will contain the conda_env file. Right now, this will fail as you don't have this file that you are trying to access for creating environment.

Please check step 6.1 in the [bike-share notebook ](https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb)for reference.

Reply:
Oh, I got it, I had removed this cell becoz of a change requested by Nickolay, but it seems that I need this to avoid this error:
ValidationException: No such file or directory: /home/runner/work/azureml-examples/azureml-examples/sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/artifact_downloads/outputs/conda_env_v_1_0_0.yml.

Thank you for looking into this. I will add these back.